beast/observationmodel/observations.py

```diff
@@ -125,35 +99,33 @@ def setFilters(self, filters):
 
     def getMags(self, num, filters):
         raise Exception('Do not use as magnitudes')
-        return np.array([ self.data[tt][num] - self.distanceModulus for tt in filters])
```

Comment:
And the success of the regression test (exactly the same results after removing the distancemodulus from observatoins.py) shows that we have not using the code in observations.py that used the distancemodulus.